src/main/java/yasp/Parse.java

```diff
@@ -291,15 +295,11 @@ public void onCombatLogEntry(Context ctx, CombatLogEntry cle) {
     }
 
     @OnEntityEntered
-    public void onEntityEntered(Context ctx, Entity e) {
-        processEntity(ctx, e, false);
-    }
-    
     @OnEntityLeft
-    public void onEntityLeft(Context ctx, Entity e) {
-        processEntity(ctx, e, true);
+    public void onEntityExistenceChanged(Context ctx, Entity e) {
```

Comment:
Actually, if this is only here for cosmetics entities, you should probably just keep the OnEntityEntered and remove OnEntityLeft (assuming ward-related annotated methods will be called separately)